ie-solver/helpers.cpp

```diff
@@ -339,6 +509,8 @@ int parse_input_into_config(int argc, char** argv, ie_solver_config * config) {
           config->boundary.reset(new Squiggly());
         } else if (!strcmp(argv[i + 1], "ELLIPSES")) {
           config->boundary.reset(new Ellipses());
+        } else if (!strcmp(argv[i + 1], "ANNULUS")) {
```

Comment:
address TODO and also change output BC (currently single electron in stokes case)